package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "reresource",
-  "version": "1.0.0-rc.7",
+  "version": "1.0.0-rc.8",
```

Comment:
About the versioning I didn't update any version yet but this would make sense for any single change. Although since the repo is still gaining shape I didn't do it so far but which approach should we take for this?

Reply:
There's no need to update. I just do it to test with npm

Reply:
OK I see. But still we should enforce an update of the version if any code was changed at least IMO

Reply:
After the first release, I will